src/main/java/eu/proteus/producer/tasks/ProteusStreamingTask.java

```diff
@@ -1,5 +1,16 @@
 package eu.proteus.producer.tasks;
```

Comment:
Why do you change all the indentation in the file? It is very difficult to review the code if all the file content changes.